File: Erik_Vincent/Assignment1/Assignment1/Model/QL/AST/Expression/LessThan.cs
namespace Assignment1.Model.QL.AST.Expression
{
    public class LessThan : Binary
    {
        public LessThan(IExpression left, IExpression right) : base(left, right) { }

        public override void Accept(IExpressionVisitor visitor) => visitor.Visit(this);
    }
}

File: piotr/org.uva.sc.piotr.ql/src/main/java/ast/model/expressions/binary/comparision/LessThan.java
package ast.model.expressions.binary.comparision;

import ast.model.expressions.Expression;
import ast.model.expressions.binary.BinaryExpression;
import ast.visitors.ASTNodeVisitor;

public class LessThan extends BinaryExpression {
    public LessThan(Expression leftSide, Expression rightSide, MetaInformation metaInformation) {
        super(leftSide, rightSide, metaInformation);
    }

    public LessThan(Expression leftSide, Expression rightSide) {
        super(leftSide, rightSide);
    }

    @Override
    public <T> T accept(ASTNodeVisitor<T> visitor) {
        return visitor.visit(this);
    }}
